refresh.template.py

```diff
@@ -221,12 +279,15 @@ def _all_platform_patch(compile_args: typing.List[str]):
 
     return list(compile_args)
 
+def _windows_platform_patch(compile_args: typing.List[str]):
+    if platform.system() == "Windows":
+        return list(arg[0] + arg[1:].replace("/", "\\") for arg in compile_args)
+    return compile_args
```

Comment:
It's not obvious to me what this does!

Reply:
Not sure why I did the `arg[0] + arg[1:]` dance. Might have been an experiment. Removed.

Reply:
Oh, wow. Thanks. I'd totally misunderstood this the first time.
Does it indeed break without this? (In my memory, Windows started supporting / in 7, and Bazel doesn't support older than that. But I could be wrong.)

Reply:
I realized also: Should we be detecting a windows command via the compiler name, as you did above, rather than detecting the host platform? Might future proof us better for, for example, cross-compilation and remote builds.

Reply:
> Does it indeed break without this?

Removed and still works. I might have introduce it because the `/` also end up in the `compile_commands.json`. But `clangd` seems to cope with this aswell. Removing this.

Reply:
Super good for me to know--and great that Window's and clangd's forward path support eliminates the need to handle this case. Thank you!